### ISSUE TITLE
Uncaught Error: ~/projects/第一章.md does not belong to repo null

### ISSUE THREAD

**Issue by tigerneil** [atom/github#1141]
lost all my draft. can't recover now. any comments?

[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.19.4 x64
**Electron**: 1.6.9
**OS**: Mac OS X 10.12.3
**Thrown From**: [github](https://github.com/atom/github) package 0.3.4-1

### Stack Trace

Uncaught Error: /Users/Tiger/projects/第一章.md does not belong to repo null

```
At /private/var/folders/pj/hpzc4r7j4fn32168szn7pfjh0000gn/T/AppTranslocation/23CBC90D-F328-4B78-A9B5-18E0F206846F/d/Atom.app/Contents/Resources/app/node_modules/github/lib/controllers/root-controller.js:807

Error: /Users/Tiger/projects/第一章.md does not belong to repo null
    at RootController.viewChangesForCurrentFile (/private/var/folders/pj/hpzc4r7j4fn32168szn7pfjh0000gn/T/AppTranslocation/23CBC90D-F328-4B78-A9B5-18E0F206846F/d/Atom.app/Contents/Resources/app/node_modules/github/lib/controllers/root-controller.js:807:19)
    at RootController.viewStagedChangesForCurrentFile (/private/var/folders/pj/hpzc4r7j4fn32168szn7pfjh0000gn/T/AppTranslocation/23CBC90D-F328-4B78-A9B5-18E0F206846F/d/Atom.app/Contents/Resources/app/node_modules/github/lib/controllers/root-controller.js:816:16)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/private/var/folders/pj/hpzc4r7j4fn32168szn7pfjh0000gn/T/AppTranslocation/23CBC90D-F328-4B78-A9B5-18E0F206846F/d/Atom.app/Contents/Resources/app/src/command-registry.js:265:35)
    at CommandRegistry.handleCommandEvent (/private/var/folders/pj/hpzc4r7j4fn32168szn7pfjh0000gn/T/AppTranslocation/23CBC90D-F328-4B78-A9B5-18E0F206846F/d/Atom.app/Contents/Resources/app/src/command-registry.js:3:65)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/private/var/folders/pj/hpzc4r7j4fn32168szn7pfjh0000gn/T/AppTranslocation/23CBC90D-F328-4B78-A9B5-18E0F206846F/d/Atom.app/Contents/Resources/app/src/command-registry.js:166:25)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchContextMenuCommand (/private/var/folders/pj/hpzc4r7j4fn32168szn7pfjh0000gn/T/AppTranslocation/23CBC90D-F328-4B78-A9B5-18E0F206846F/d/Atom.app/Contents/Resources/app/src/atom-environment.js:1344:34)
    at EventEmitter.outerCallback (/private/var/folders/pj/hpzc4r7j4fn32168szn7pfjh0000gn/T/AppTranslocation/23CBC90D-F328-4B78-A9B5-18E0F206846F/d/Atom.app/Contents/Resources/app/src/application-delegate.js:347:31)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
```

### Commands

```
 33x -0:20 core:undo (div.markdown-preview-enhanced.native-key-bindings)
     -0:07.5.0 github:view-staged-changes-for-current-file (div.line)
```

### Non-Core Packages

```
atom-django 0.3.2 
atom-material-ui 1.3.10 
autoclose-html 0.23.0 
autocomplete-python 1.8.63 
busy-signal 1.4.1 
d3-snippets 1.0.2 
diagrams 0.26.4 
graphviz-preview 1.7.0 
graphviz-preview-plus 1.3.0 
intentions 1.1.2 
javascript-snippets 1.2.1 
language-dot 1.7.0 
language-latex 1.0.0 
language-markdown 0.22.0 
latex 0.43.0 
latexer 0.3.0 
linter 2.1.4 
linter-ui-default 1.2.4 
MagicPython 1.0.10 
markdown-img-paste 0.3.9 
markdown-pdf 1.5.0 
markdown-preview-enhanced 0.12.3 
markdown-writer 2.6.5 
python-tools 0.6.9 
sequence-diagram 0.0.2 
slides-preview 0.3.1 
terminal-plus 0.14.5 
textmate-django-dark-syntax 0.1.0 
```

**Comment by 50Wliu** [comment]
This has been fixed in Atom 1.20, currently in beta.

**Comment by tigerneil** [comment]
thanks
